### PR TITLE
npm package of 3.3.4

### PR DESCRIPTION
**Pull request by mbarnes-sb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ],
+    "path": [
+      "win64",
+      "win64/**",
+      "macos",
+      "macos/**"
+    ]
+  },
+  "scripts": {
+    "install": "prades install"
```
Comment (krunal-patel-fl4re) [inline package.json]: You probably want to lock down the prades version like we discussed a while back for one of the other externals.  It was the agreed upon protocol as decided by you and Remi.  Surprised to see that even starvr-sdk doesn't do this yet.  I don't recall which module does it correctly.

File: publish_binary_on_windows.bat [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+git rev-parse --is-inside-work-tree
+if errorlevel 1 exit /b %errorlevel%
+
+call npm install
+if errorlevel 1 exit /b %errorlevel%
+
+call npm --no-git-tag-version version patch
+if errorlevel 1 exit /b %errorlevel%
+
+call node_modules\.bin\prades publish -v
```
Comment (krunal-patel-fl4re) [inline publish_binary_on_windows.bat]: again i feel like we locked prades down.

**Comment by mbarnes-sb** [comment]
build test looks ok

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@sb/ffmpeg",
+  "version": "3.3.4-dev1",
```
[the remark this answers is not in the document]
Comment (mbarnes-sb) [reply package.json]: using http://semver.org/#spec-item-9

File: package.json (hunk 1 above)
Comment (mbarnes-sb) [reply package.json]: prades tool is not a dependency.

**Pull request merged**